metadata/jobs.xml

```diff
@@ -15,8 +15,8 @@
             <run-recurring enabled="true">
                 <recurrence>
                     <date-from>2023-01-01Z</date-from>
-                    <start-time>14:00:00.000Z</start-time>
-                    <interval>1d</interval>
+                    <start-time>00:00:00.000Z</start-time>
```

Comment:
Updated to be consistent with other jobs, but please let me know if that might interfere with other code elsewhere.